modules/README.md

```diff
@@ -9,63 +9,34 @@ $ cmake -D OPENCV_EXTRA_MODULES_PATH=<opencv_contrib>/modules -D BUILD_opencv_<r
 ```
 
 - **aruco**: ArUco and ChArUco Markers -- Augmented reality ArUco marker and "ChARUco" markers where ArUco markers embedded inside the white areas of the checker board.
-
 - **bgsegm**: Background segmentation algorithm combining statistical background image estimation and per-pixel Bayesian segmentation.
-
```

Comment:
You need to:
- remove unrelated changes
- squash your commits
- rebase on upstream "master" branch (last two steps can be don via "git rebase -i upstream/master", where upstream is configured remote for opencv/opencv_contrib (see GitHub help))

Reply:
Thanks a lot! Now everything works out!!! 👍